core/templates/components/forms/schema-based-editors/schema-based-float-editor.directive.ts

```diff
@@ -120,7 +120,7 @@ angular.module('oppia').directive('schemaBasedFloatEditor', [
             ctrl.labelForErrorFocusTarget =
               FocusManagerService.generateFocusLabel();
             if (ctrl.localValue === undefined) {
-              ctrl.localValue = 0.0;
+              ctrl.localValue = null;
```

Comment:
Hi @ReshuKumari, just one question, based on git blame (see [commit history](https://github.com/oppia/oppia/commits/develop/core/templates/components/forms/schema-based-editors/schema-based-float-editor.directive.ts)), it looks like this particular logic hasn't changed in a long time. Considering that this is a regression (i.e. it works fine in the currently deployed release branch `release-3.1.2` but fails on develop), we should identify the change that caused this to make sure that our fix is fool-proof. Could you try git-bisect to identify the commit that caused this please? Thanks!

Reply:
Hi @ReshuKumari I took a look and it looks a regression introduced in https://github.com/oppia/oppia/pull/12753. It seems to be the case that, we are incorrectly treating `localValue` as an Output / event in the HTML. See here:

https://github.com/oppia/oppia/blob/2351a67ccec20809fe8a21025a31f3ab3120ed22/extensions/interactions/NumericInput/directives/numeric-input-interaction.component.html#L3-L7

It should be `[localValue]` and not `(localValue)`. The `(..)` syntax is reserved for event binding.


Could you undo your changes in the PR and fix `numeric-input-interaction.component.html` as mentioned above? Thanks!

Reply:
Thanks for taking a look at it addressed in pr [13560](https://github.com/oppia/oppia/pull/13560)